lib/inputs/binaries/static/index.ts

```diff
@@ -1 +1,24 @@
-// TODO
+import { ExtractAction, ExtractedLayers } from "../../../extractor/types";
+import { streamToHash } from "../../../stream-utils";
+
+export const getNodeBinariesFileContentAction: ExtractAction = {
+  actionName: "node",
+  fileNamePattern: "**/node",
+  callback: streamToHash,
+};
+
+export function getBinariesHashes(extractedLayers: ExtractedLayers): string[] {
+  const hashes: string[] = [];
+  for (const fileName of Object.keys(extractedLayers)) {
+    for (const actionName of Object.keys(extractedLayers[fileName])) {
+      if (actionName !== getNodeBinariesFileContentAction.actionName) {
+        continue;
+      }
+      if (!(typeof extractedLayers[fileName][actionName] === "string")) {
+        throw new Error("expected string");
+      }
+      hashes.push(extractedLayers[fileName][actionName] as string);
```

Comment:
what happens if the same file in the same path occurs in multiple layers?

Reply:
the extraction logic always provides the file in the path from the most recent (relevant) layer,
so theoretically we don't expect duplications here.
that said, I don't mind putting it explicitly using a ```set```.
also enhanced the test to check for it.